vsts/pipelines/templates/_releaseBaseImagesStepTemplate.yml

```diff
@@ -29,43 +29,13 @@ steps:
     SourceFolder: '$(System.ArtifactsDirectory)'
     TargetFolder: '$(Build.ArtifactStagingDirectory)'
 
-- task: ms-devlabs.utilitytasks.task-Shellpp.Shell++@0
-  displayName: 'Pull and create release tags for staging ACR'
-  inputs:
-    type: FilePath
-    scriptPath: ./vsts/scripts/tagBaseImagesForRelease.sh
-    args: '${{ parameters.baseImageName }} ${{ parameters.acrProdName }}'
-
 - task: ms-devlabs.utilitytasks.task-Shellpp.Shell++@0
   displayName: 'Pull and create release tags for PME staging ACR'
```

Comment:
so are we now only push to pme ACRs and will remove all non-pme ACRs?

Reply:
please make sure those build/runtime images under old ACRs will not be used in the future

Reply:
we dont use them anymore ... it was just sitting there for no reason